code/game/objects/structures/window.dm

```diff
@@ -27,7 +27,23 @@
 
 /obj/structure/window/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>Alt-click to rotate it clockwise.</span>")
+	if(reinf)
+		if(anchored && state == WINDOW_SCREWED_TO_FRAME)
+			to_chat(user, "<span class='notice'>The window is <b>screwed</b> to the frame.</span>")
```

Comment:
Well it's talking about... the window inside the frame unit? The tool use messages don't use [src] either, should I change that? I'm not entirely clear of what the window object actually represents when its reinforced.